qemu/tests/hv_type.py

```diff
@@ -16,11 +18,27 @@ def run(test, params, env):
     param env: the test env object
     """
     timeout = params.get_numeric("timeout", 360)
-
+    virt_what_remove_cmd = params["virt_what_remove_cmd"]
+    virt_what_pkg = params["virt_what_pkg"]
+    virt_what_guest_dir = params["virt_what_guest_dir"]
+    virt_what_install_cmd = params["virt_what_install_cmd"]
     error_context.context("Boot the guest with all flags", logging.info)
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     session = vm.wait_for_login(timeout=timeout)
+    # use virt-what-1.18-6.el8.x86_64 for testing
```

Comment:
Shall we need check the virt_what package has been installed inside guest? then uninstall it.

Reply:
Actually it will be install default, but add related steps will be Logically rigorous.
I will modify it, thanks

Reply:
(1/1) Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.hv_type.q35: PASS (68.86 s)